draft-ietf-rats-corim.md

```diff
@@ -2136,71 +2121,44 @@ Hence, the Verifier may need to maintain multiple Attestation Results contexts.
-indicates that the authority named by `measurement-map`/`authorized-by`
-asserts that the actual state of one or more Claims within the
-Target Environment, as identified by `environment-map`, have the
-measurement values in `measurement-map`/`mval`.
+Each ACS ECT entry SHALL contain `authority`.
+See {{sec-authority}}.
+There can be multiple entries in the ACS which have the same `environment-map` and a different authority.
 
-ECT authority is represented by cryptographic keys. Authority
-is asserted by digitally signing a Claim using the key. Hence, Claims are
-added to the ACS under the authority of a cryptographic key.
+Each ACS entry is an ECT Claim.
+The ECT `environment` and `element-id` together define the Claim's name.
+The ECT `element-claims` are the Claim's *actual state*.
 
-Each Claim is encoded as an ECT. The `environment-map` and a
-key within `measurement-values-map` encode the name of the Claim.
-The value matching that key within `measurement-values-map` is the actual
-state of the Claim.
-
-This specification does not assign special meanings to any Claim name,
-it only specifies rules for determining when two Claim names are the same.
-
-If two Claims have the same `environment-map` encoding then this does not
-trigger special encoding in the Verifier. The Verifier follows instructions
-in the CoRIM file which tell it how claims are related.
-
-If Evidence or Endorsements from different sources has the same `environment-map`
-and `authorized-by` then the `measurement-values-map`s are merged.
+This specification defines rules for determining when two Claim names are equivalent.
+See {{sec-compare-environment}} and {{sec-compare-element-map}}.
+Equivalence typically means values MUST be binary identical.
 
-The ACS must maintain the authority information for each ECT. There can be
-multiple entries in `state-triples` which have the same `environment-map`
-and a different authority.
-See {{sec-authorized-by}}.
+If two Claims have the same Claim name, the CoRIM triples instruct the Verifier on how these Claims are related.
 
-If the merged `measurement-values-map` contains duplicate codepoints and the
-measurement values are equivalent, then duplicate claims SHOULD be omitted.
-Equivalence typically means values MUST be binary identical.
+If multiple Claims have the same name and the measurement values are equivalent, they are said to be *duplicates*.
+Measurement values are duplicates if the `measurement-values-map` codepoints and values for each are equivalent.
+Duplicate claims SHOULD be omitted.
 
-If the merged `measurement-values-map` contains duplicate codepoints and the
-measurement values are not equivalent, then a Verifier SHALL report
-an error and stop validation processing.
+If multiple Claims have the same name and `measurement-values-map` contains duplicate codepoints but the measurement values are not equivalent, then a Verifier SHALL report an error and stop validation processing.
```

Comment:
If this is intended to be different from https://www.rfc-editor.org/rfc/rfc8949.html#name-basic-validity I would not recommend it.

Reply:
It’s the duplicate key basic validity condition for maps if we’re thinking about map merging.
“name” is maybe not clear as being the combination of environment map, mkey, and authority around the measurement values map.

Reply:
earlier text defined "name" so that the verbose explanation of the combination of the environment-map and the element-id doesn't have to be repeated multiple times in subsequent text referring to "name".